releasenotes/notes/allowIntegerforQuantumRegisterSize-2e9a6cc033bc962e.yaml

```diff
@@ -0,0 +1,13 @@
+---
+prelude: >
+    allow use of type integer for for QuantumRegister size 
+
+issues:
+  - |
+    QuantumRegister allows only int, slice, list to denote size
+
```

Comment:
The prelude is only added upon release, we don't need to add this in a PR. 🙂  Also you don't need to refer to the issue, I think this section is for issues with the release and does not refer to GitHub issues.
```suggestion
```